docs/misc/hoverline.md

```diff
@@ -94,14 +94,14 @@ author: mwsht, sshwy, ouuan, Ir1d, Henry-ZHR, hsfzLZH1
     }
     ```
 
-## 最大子矩形
+## Largest submatrix
 
-???+note "[P4147 玉蟾宫](https://www.luogu.com.cn/problem/P4147)"
-    给定一个 $n \times m$ 的包含 `'F'` 和 `'R'` 的矩阵，求其面积最大的子矩阵的面积 $\times 3$ ，使得这个子矩阵中的每一位的值都为 `'F'` 。
+???+note "[P4147 Yuchan Palace](https://www.luogu.com.cn/problem/P4147) (original link in Chinese)"
+    Given a $n \times m$ matrix containing `'F'` and `'R'`, find the area $\times 3$ of the largest submatrix so that the value of each element is `'F'`.
```

Comment:
```suggestion
    Given a $n \times m$ matrix consists of`'F'` and `'R'`, find the area $\times 3$ of the largest submatrix so that the value of all elements in this matrix are `'F'`.
```